src/info.json

```diff
@@ -62,6 +62,19 @@
           "value": "claude-3-5-sonnet-20240620"
         }
       ]
+    },
+    {
+      "identifier": "user_translation_prompt",
+      "type": "text",
+      "title": "Claude提示词",
+      "desc": "使用${sourceLanguage}表示源语言，${targetLanguage}表示目标语言"
+    },
+    {
+      "identifier": "user_sys_prompt",
```

Comment:
I do not believe it is a good decision to make system prompts perceptible to the average user, particularly when user prompt customization options are also provided, It will only benefits developers. A better approach may be to only provide user prompt customization option.

Reply:
Thank you for reviewing the code. However, some users might attempt to use this plugin for more advanced purposes, such as utilizing Claude as a quick coding assistant. Therefore, I believe it is necessary to allow users to customize both system prompts and user prompts simultaneously

Reply:
Maybe user prompt is also capible of doing those customizations, refer to [the documentation of system prompt](https://docs.anthropic.com/en/docs/build-with-claude/prompt-engineering/system-prompts#financial-analysis-without-role-prompting), Anthropic also set the role in the user prompt in the "Examples" section.

Reply:
Yeah, user prompts can indeed define Claude's identity, similar to how you interact with Claude in Claude.ai. System prompts seem to be primarily used for conversations with context, and Claude permanently records the system prompt. I'm currently modifying the code and will submit another commit shortly.